### PR TITLE
clean up address verify property and some miscellaneous request logic

### PR DESCRIPTION
**Pull request by Roguelazer**
allows users to just pass `True` for verify and verify_strict instead of having to pass `[True]`. Also properly quotes in case a user accidentally passes something icky.

as always, I am a firm believer that functions should be functions, not classmethods

### CODE REVIEW

File: easypost/__init__.py [hunk 1]
```diff
@@ -634,13 +644,15 @@ def create(cls, api_key=None, verify=None, verify_strict=None, **params):
...
-            )
+        verify_params = {}
+        for key, value in (('verify', verify), ('verify_strict', verify_strict)):
+            if not value:
+                continue
+            elif isinstance(value, (bool, str)):
+                value = [value]
+            verify_params[key] = value
+        if verify_params:
+            url += '?' + _urlencode_list(verify_params)
```
[the remark this answers is not in the document]
Comment (Roguelazer) [reply easypost/__init__.py:655]: frankly I think we should just switch to encoding the bodies with JSON and get rid of all of this (absolutely awful) urlencoding crap

**Review by jontsai — DISMISSED** [review]
LGTM

**Review by Justintime50 — DISMISSED** [review]
lftm based on our convos reviewing. Looks like this will need to be rebased against master?

**Comment by Roguelazer** [comment]
rebased

File: easypost/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pseudoben) [reply easypost/__init__.py:655]: I strongly agree with moving away from URL-encoding request parameters.

@jontsai putting an adapter in front of the HTTP request logic is a good idea, whether or not we stick with URL encoding parameters. In addition to that, I would love to drop support for `urlfetch`, as it's not in the GAE Python 3 runtime. Sadly, however, Google has stated that [they will continue to support the 2.7 runtime](https://cloud.google.com/python/docs/python2-sunset#appengine-python2) indefinitely, so it's not clear when we'll be able to do that.

**Review by jontsai — APPROVED** [review]

**Pull request merged**